doc/source/ray-observability/state/ray-state-api-reference.rst

```diff
@@ -3,7 +3,9 @@ Ray State API
 
 .. _state-api-ref:
 
-.. tip:: APIs are pre-alpha and under active development. APIs are subject to change and not stable across versions.
+.. note:: 
+
```

Comment:
can you link to the definition of alpha here (api-stability) and reduce verbosity?